src/main/resources/db/changelog/changelog.sql

```diff
@@ -1098,3 +1098,7 @@ ALTER TABLE `AdherenceRecords`
 ADD COLUMN `postProcessingAttributes` text COLLATE utf8_unicode_ci DEFAULT NULL,
 ADD COLUMN `postProcessingCompletedOn` bigint(20) unsigned DEFAULT NULL,
 ADD COLUMN `postProcessingStatus` varchar(255) DEFAULT NULL;
+
+-- changeset bridge:81
+ALTER TABLE `Assessments`
+ADD COLUMN `phase` enum('DRAFT', 'REVIEW', 'PUBLISHED') DEFAULT 'DRAFT',
```

Comment:
Shared assessments are published by default, aren't they?

Reply:
Good catch. Just added code to set Shared assessments to published when they are created. Not sure the best approach for migrating existing shared assessments.